### ISSUE TITLE
Traning -- why do we need av-benchmark?

### ISSUE THREAD

**Issue by bliinmaker** [hkchengrex/MMAudio#36]
Good afternoon, tell me, I don't quite understand why I should use av-benchmark in model training.

Thank you in advance!

**Comment by hkchengrex** [comment]
Hi, 

From TRAINING.md
> "We use this library to automatically evaluate on the validation set during training, and on the test set after training."

This library is used to compute various metrics like FD, IB, and DeSync in validation and testing. They are used to gauge the model performance during and after training.
